poedit.appdata.xml

```diff
@@ -1,31 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<components version="0.11">
```

Comment:
What's the implication of removing this for older distros, if any? Won't it break parsing by some?

Reply:
Older distros have a strict policy to update applications...

Reply:
> Older distros have a strict policy to update applications…

This PR was motivated by a flatpak package update. The whole point of flatpak (for software like Poedit that is natively packaged) is deploying to older distros that don't have the latest & greatest natively packaged. Presumably such a bit older distros won't have flatpak runtime always at the latest version either and maybe the AppStream stuff is exposed to the hosting OS as well for integration.

So again: what are compatibility implications? How new must the tools be to understand `<components>`-less files?

Reply:
The appdata is used by the software center to show up the information related to the application the user wants to install. Flatpak/Flathub package will work without any issues. 
The issue here is whether that old appcenter will show up the correct appdata information or not. The example in the documentation is using the same thing as I proposed on this PR https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps which was written in 2013

Reply:
That really doesn't say much — that's some random example maybe initially written in 2013 and possibly edited since then (and the whole comment might have been made up, including the year — it's an example).

Here's a [recent file](https://github.com/ximion/appstream/blob/master/tests/samples/collection/xml/lvfs-gdpr.xml) in AppStream's repo still using `<components>`. The [current schema](https://github.com/ximion/appstream/blob/master/data/appstream.xsd) for the file format still allows for it.

But the schema was also last modified in 2014, which shows that not using `<components>` was valid back then. That's old enough for me.

It brings another question, though: why change this bit and remove `<components>`? What was the motivating reason?

Reply:
Never said that it was outdated or deprecated. It's just that your application ships only one component. I can revert that part of course!

Reply:
Well, you did: "This PR updates the AppData **with the latest requirements**", so I operated under the assumption it is needed (because PR description and commit messages are truthful and one-thing-per-one-commit).

It does make sense to clean this up as you did, but it's probably more suitable as a separate commit (within a single PR).

Reply:
Alright, will take care of that this evening with the rest of the comments! thanks for the review :+1: